exercises/concept/stage-heralding/.docs/instructions.md

```diff
@@ -0,0 +1,108 @@
+# Instructions
+
+You are a stage herald at an international conference.
+It's your job to announce speakers, remind them when their time is up, and moderate the Q&A session.
+You're given a list of speakers on your stage in advance and need to turn them into moderation cards.
+
+Since the list follows a particular formatting, you decide to write a short program that parses the list you're given using regular expressions (regex) and automatically generates the moderation cards.
+
+!!! note
+    This exercise is **not** about learning regular expressions.
+    It's about learning to use regex within Julia.
+    The hints contain a regular expression that you can use if you don't want to write one yourself.
+
+The list you're given contains the following information for each talk:
+
+- its title,
+- the speaker's name,
+- their pronouns (optional),
+- their organisation (optional),
+- the start time of the talk,
+- the start of the Q&A session (optional),
+- and the end time of the talk.
+
+A full entry has the following format:
+
+```text
+»Speedrunning 101« – Sasha Duda Krall (they/them), from GDQU. Start: 13:00, Q&A: 13:20, End: 13:30
+```
+
+Here, the title is `Speedrunning 101`, the speaker is called `Sasha Duda Krall`, from `GDQU`, and uses `they/them` pronouns.
+The talk starts at `13:00`, there will be a Q&A starting at `13:20`, and the entire session ends at `13:30`.
+
+!!! note
+    Since names may contain regular dashes `-`, an en dash `–` is used to separate the speaker and talk name.
+
+The moderation card that your program should generate for that talk looks like this:
+
+```text
+- Our next speaker is Sasha Duda Krall, from GDQU
+- Their talk is called »Speedrunning 101«
+- They will answer your questions in the Q&A session at the end of the talk, starting at 13:20
+
+13:00 - 13:20 - 13:30
+```
+
+The entire list may look like this:
+
+```text
+»Speedrunning 101« – Sasha Duda Krall (they/them), from GDQU. Start: 13:00, Q&A: 13:20, End: 13:30
+»How I learned to say Farewell« – Madeline (she/her). Start: 13:40, Q&A: 14:00, End: 14:05
+»How To Find a Good Title For Your Conference Talk« – Vítězslav Kruse (he/him). Start: 14:15, End: 14:40
+»Why Emoji Matter❣« – Ash van Amelsvoort, from the University of 🧬🧪⚛. Start: 14:50, Q&A: 15:05, End: 15:10
```

Comment:
```suggestion
»Why Emoji Matter❣« – Ash van Amelsvoort, from the University of 🧬🧪⚛. Start: 14:50, Q&A: 15:05, End: 15:10
»Can dogs look up?« – Kira "k1ralli" Sørensen. Start: 20:50, Q&A: 21:05, End: 21:10
»SQL Injection 201« – Robert'); DROP TABLE Students; --. Start: 20:50, Q&A: 21:05, End: 21:10
```

Reply:
We can have an example with a comma in it, but then the student has to solve it with non-greedy matches (which I think is the right way, but opinions may vary).

Reply:
Imo that complicates the regex a bit too much for people that are newish to regex. I'd prefer the structural punctuation to be unique.